specification/cognitiveservices/data-plane/QnAMaker/stable/v4.0/QnAMakerRuntime.json

```diff
@@ -519,6 +519,20 @@
       "x-nullable": true,
       "description": "Post body of the request.",
       "x-ms-parameter-location": "method"
+    },
+    "EndpointKeyHeader": {
+      "name": "Authorization",
+      "in": "header",
+      "description": "Authorization header with Primary/Secondary Endpoint Key of Runtime (for example: EndpointKey 8739caf6-8d95-4395-a020-904ae576f7c4)",
+      "required": true,
+      "type": "string"
+    },
+    "ContentTypeHeader": {
+      "name": "Content-Type",
```

Comment:
sdk automatically adds these, not sure but this might break sdk
 _httpRequest.Content.Headers.ContentType =System.Net.Http.Headers.MediaTypeHeaderValue.Parse("application/json; charset=utf-8");